sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubProducerClient.cs

```diff
@@ -358,9 +361,9 @@ public virtual Task<string[]> GetPartitionIdsAsync(CancellationToken cancellatio
         /// <seealso cref="SendAsync(IEnumerable{EventData}, SendOptions, CancellationToken)" />
         /// <seealso cref="SendAsync(EventDataBatch, CancellationToken)" />
         ///
-        public virtual Task SendAsync(EventData eventData,
-                                      SendOptions options,
-                                      CancellationToken cancellationToken = default)
+        internal virtual Task SendAsync(EventData eventData,
```

Comment:
Just curious, why are we keeping these methods and not removing them ?

Reply:
Because they are expected to return post-GA.   There was some disagreement on what the behavior should be, so they've been removed from the GA scope.  The internals may change a bit but the core parts and signature are expected to be reasonably stable.   

Between that and the amount of churn that it would cause across our test surface, I feel that its better to internalize them than to rip them out only to add them back.